### PR TITLE
update wording on DCB port config

### PR DESCRIPTION
**Pull request by mmahacek**
Feedback on docs that there is some ambiguity if this command is required (it's not)

### All Contributors

* [X] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### External References

* JIRA (Issue Tracker): No Jira

### CODE REVIEW

**Review by Bonrob2 — APPROVED** [review]
This might take me a while to review. So many words! (Thanks.)

**Review by emarsh1995 — APPROVED** [review]

**Comment by mmahacek** [comment]
Got some additional feedback from client services...

File: docs/modules/operation/pages/deep-dive/device-config-backup/configuration.adoc [hunk 1]
```diff
@@ -23,9 +23,10 @@ ssh -p 8201 admin@localhost
 [source, karaf]
 ----
 config:edit org.opennms.features.deviceconfig.tftp
-config:property-set port 10069
+config:property-set port 10069 <1>
 config:update
 ----
+<1> If you have set `CAP_NET_BIND_SERVICE` on the service unit, you can set the port to the tftp default of `69` and skip the firewall port redirect steps below.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/device-config-backup/configuration.adoc:29]:
```suggestion
<1> If you have set `CAP_NET_BIND_SERVICE` on the service unit, you can set the port to the TFTP default of `69` and skip the firewall port redirect steps below.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Pull request merged**